### ISSUE TITLE
Xls open_workbook throws error: Expected "little-endian" marker, found \'\\xff\\xff\'\n'

### ISSUE THREAD

**Issue by Alchemication** [python-excel/xlrd#118]
I'm really stuck on this for a good couple of days now, looks like line 91 is causing an issue (file1 attached), unfortunately I can't modify the source "xls" feed as it comes from SAP Business Objects with suspicious 1 empty (sort of stretched vertically) row at the end of file (can't attach a sample file as it contains user data and after truncating it - script starts to work fine again),

Am I missing something? Is this something that can be patched?

My code is simple (initially):
`wb = xlrd.open_workbook(source)`

![file1](https://cloud.githubusercontent.com/assets/2961584/4320852/f8fe59f4-3f3a-11e4-8992-4d08c062cf57.png)

**Comment by Alchemication** [comment]
Hi, Apologies that it took me so long. Was assigned to another project.

Now, I've commented out the lines responsible for checking for that marker and it works (also it's blazing super-fast which is amazing!!). Not sure what to do now. Obviously I did the change only on my machine, but to get it up on my server it needs to be done properly. Any recommendations? What is that check doing in the first place?

Thanks a lot!
